tests/test_pytest_mock.py

```diff
@@ -310,6 +310,9 @@ def bar(self, x):
     assert spy.spy_return == 30
     assert spy.spy_exception is None
 
+    # Testing spy can still be reset.
+    mocker.resetall()
+
```

Comment:
Found a working test case. If you add this line to the previous version `3.6.0` then it triggers the error.